app/main.py

```diff
@@ -1 +1,61 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self,
+                 name: str
+                 ) -> None:
```

Comment:
It's my personal taste, but this kind of code-style for functional parameters are not so great.
I suggest 2 other options:
1st:
Simply in 1 line, because we have only 2 parameters here (no need to make a new lines):
`def __init__(self, name: str) -> None:`
2nd:
if a lot of arguments - you can use `black` codestyle:
```python
def __init__(
    self,
    param1,
    param2,
    param3,
    param4,
):
    pass
```

Reply:
okay, thanks for the advice!) Is second option usable only for __init__ or even methods and functions?